packages/amazonq/test/unit/amazonqFeatureDev/util/files.test.ts

```diff
@@ -56,6 +64,35 @@ describe('file utils', () => {
             assertTelemetry('amazonq_bundleExtensionIgnored', { filenameExt: 'mp4', count: 1 })
         })
 
+        it('should ignore devfile.yaml when setting is disabled', async function () {
+            const workspace = await getWorkspaceWithDevFile()
```

Comment:
One last thing, I think we need to mock here as well. Otherwise it will pick up whatever value the user has already set I think

Reply:
that's fine, user will have nothing by default, I don't think there is a point of mocking here

Reply:
But what if a dev runs the test and has the setting already set? Won't it have the potential to fail the test since its running against the users setting?